docs/docs/dev_docs/getting_started/quickstart.md

```diff
@@ -57,7 +65,9 @@ To install a specific version of the sandbox, you can set the environment variab
 SANDBOX_VERSION=<version> /bin/bash -c "$(curl -fsSL 'https://sandbox.aztec.network')"
 ```
 
-NOTE: If `SANDBOX_VERSION` is not defined, the script will pull the latest release of the sandbox.
+NOTE: If `SANDBOX_VERSION` is not defined, the script will pull the latest release of the sandbox. The sandbox version should be the same as your `@aztec/cli` package to ensure compatibility.
+
+Once docker is up, you can see ethereum layer 1 activity through the local (otterscan)[http://localhost:5100]. This is especially useful for dapps that use L1-L2 messaging through [portal contracts](../contracts/portals/main.md).
```

Comment:
I think the ordering of ()[] is wrong on the otterscan link. Should it not be `[otterscan](http://localhost:5100)` to be rendered correctly?